src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/Helpers/NativeSnackBar.ios.macos.cs

```diff
@@ -99,9 +100,45 @@ public NativeSnackBar Show()
 				Dismiss();
 			});
 
+			AddDismissGestures();
+
 			return this;
 		}
 
+		private void AddDismissGestures()
+		{
+#if __IOS__
+			if (SnackBarView == null)
+				return;
+
+			var tapGesture = new UITapGestureRecognizer();
+			tapGesture.AddTarget(OnTap);
+			SnackBarView.AddGestureRecognizer(tapGesture);
+
+			var horizontalSwipeGesture = new UISwipeGestureRecognizer();
+			horizontalSwipeGesture.Direction = UISwipeGestureRecognizerDirection.Left | UISwipeGestureRecognizerDirection.Right;
+			horizontalSwipeGesture.AddTarget(OnSwipe);
+			SnackBarView.AddGestureRecognizer(horizontalSwipeGesture);
+
+			var verticalSwipeGesture = new UISwipeGestureRecognizer();
+			verticalSwipeGesture.Direction = UISwipeGestureRecognizerDirection.Down | UISwipeGestureRecognizerDirection.Up;
+			verticalSwipeGesture.AddTarget(OnSwipe);
+			SnackBarView.AddGestureRecognizer(verticalSwipeGesture);
+#endif
+		}
+
+		private void OnTap()
```

Comment:
This code has no references on macOS.
Why is it only for iOS? Can we add the same for macOS?